### ISSUE TITLE
Server manager

### ISSUE THREAD

**Issue by Krzmbrzl** [mumble-voip/mumble#5924]
### Context

Managing a single server or managing multiple virtual servers running within the same Mumble server instance.

### Description

Create a small separate, stand-alone GUI application that can be used to configure a running server instance. This should include editing the channel tree and also setting config options.

As a bonus it would be nice to be able to manipulate groups and ACLs from there as well.

### Mumble component

Server

### OS-specific?

No

### Additional information

See also
- #5913
- #4183
- #4164
- #2569

**Comment by trymeouteh** [comment]
Would like to see this for Linux for sure, would be nice to see this on Windows too.

**Comment by Krzmbrzl** [comment]
Yeah I think if we implement it, we should do it in a cross platform way so that it is available on all platforms Mumble itself can be run on.

**Comment by JobberRT** [comment]
> 

how about a simple web page?

**Comment by Krzmbrzl** [comment]
I think that using a web page as frontend probably makes things more complicated, unless we do a web only version, which uses something like PHP to interface with the server via Ice.
But personally, I wouldn't want to tinker with a web only solution. I'm just not into web dev at all :shrug:

That's not to say that it wouldn't be a viable solution, in general.

**Comment by trymeouteh** [comment]
By running this command, wouldn't this allow you to run multiple servers in the terminal window and be able to stop/kill a server by pressing CTRL+C?

To run multiple servers, you will need to change the INI path and have a separate terminal tab or window for each server.

`mumble-server -fg -ini "/home/USERNAME/Services/Mumble Servers/SERVER NAME.ini"`

**Comment by Krzmbrzl** [comment]
What command?

**Comment by trymeouteh** [comment]
> What command?

```
mumble-server -fg -ini "/home/USERNAME/Services/Mumble Servers/SERVER NAME.ini"
```

**Comment by Krzmbrzl** [comment]
Ah :bulb:
Yes indeed that works. However, the main thing of a server manager (imo) would be to be able to also take care of the server's configuration and user/channel administration, which can currently only be done via the Ice interface.
